### PR TITLE
#Calculator Adjusted my HTML and CSS files as requested by Ebenezer Don. 

### PR DESCRIPTION
**Pull request by Favour-dgreat**
My Pull Request

### CODE REVIEW

**Comment by Favour-dgreat** [comment]
Hope I got this right?

**Comment by ebenezerdon** [comment]
@Favour-dgreat please provide a description for this PR.

**Comment by ebenezerdon** [comment]
@Favour-dgreat your pull request name should also describe what's in it.

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    border:solid 2px #C70039;
+}
+.header p{
+    margin-left: 10%;
+    color:white;
+    font-size: 22px;
+    font-weight:bold;
+}
+
+
```
Comment (ebenezerdon) [inline styles.css]: Please remove this empty line.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```
Comment (ebenezerdon) [inline index.html]: You're mixing both 4-spaced and 2-spaced indentation. Please stick to one.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Calculator</title>
+    <link href="styles.css" type="text/css" rel="stylesheet">
+</head>
+<body>
+  <div class="header">
+    <p>Favour's Calculator</p>
+
```
Comment (ebenezerdon) [inline index.html]: Remove this empty line.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,82 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700,800,900&display=swap');
+*
+{
+    margin:0;
+    padding:0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+}
```
Comment (ebenezerdon) [inline styles.css:8]: Add an empty line after each class or tag is styled.

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    font-family: 'Poppins', sans-serif;
+}
+body{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 100vh;
+    background:#091921;
+}
+.header{
```
Comment (ebenezerdon) [inline styles.css]: Please add a space after the class name and just before the opening curly brace;
this: `.header {` instead of `.header{`

**Comment by ebenezerdon** [comment]
See this: https://namingconvention.org/git/pull-request-naming.html

**Review by ebenezerdon — CHANGES_REQUESTED** [review]
Let me know when you've worked on the requested changes.

File: styles.css (hunk 4 above)
Comment (Favour-dgreat) [reply styles.css:8]: Done

**Review by Favour-dgreat — COMMENTED** [review]
Done

**Review by Favour-dgreat — COMMENTED** [review]
I think I am done

**Comment by Favour-dgreat** [comment]
Hope all is okay now?

**Comment by ebenezerdon** [comment]
> See this: https://namingconvention.org/git/pull-request-naming.html

@Favour-dgreat did you go through this?

**Comment by Favour-dgreat** [comment]
I went through the link, but didn't understand.

**Comment by Chiagozielam** [comment]
Great work Favour. Your design didn't quite match the original design given for replication. One thing to note is to use the right color scheme while implementing.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+<input class="value" type="text" name="txt" readonly="">
+<span class="num clear" onclick="document.calc.txt.value =''">c</span>
+<span class="num" onclick="document.calc.txt.value +='/'">/</span>
+<span class="num" onclick="document.calc.txt.value +='*'">*</span>
+<span class="num" onclick="document.calc.txt.value +='7'">7</span>
+<span class="num" onclick="document.calc.txt.value +='8'">8</span>
+<span class="num" onclick="document.calc.txt.value +='9'">9</span>
+<span class="num" onclick="document.calc.txt.value +='-'">-</span>
+<span class="num" onclick="document.calc.txt.value +='4'">4</span>
+<span class="num" onclick="document.calc.txt.value +='5'">5</span>
```
Comment (Chiagozielam) [inline index.html:29]: Consider using `<button></button` tags for buttons instead of `span`

**Comment by ebenezerdon** [comment]
> Great work Favour. Your design didn't quite match the original design given for replication. One thing to note is to use the right color scheme while implementing.

Favor's submission is from last year and not part of the current challenge.

**Pull request closed without merging**